app/jobs/slack_inviter_job.rb

```diff
@@ -0,0 +1,20 @@
+require 'slack/client'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.